packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/quanticHeadlessLoader.js

```diff
@@ -99,30 +105,31 @@ async function initEngine(engineId) {
     }
     await Promise.all(dependencyPromises);
 
-    if (!window.coveoHeadless[engineId].config) {
-      throw new Error('Engine configuration has not been set.');
+    if (!window.coveoHeadless[engineId].options) {
+      throw new Error('Engine options have not been set.');
     }
-    const configuration = await window.coveoHeadless[engineId].config.promise;
-    return CoveoHeadless.buildSearchEngine({configuration})
+    const options = await window.coveoHeadless[engineId].options.promise;
+    return window.coveoHeadless[engineId].engineConstructor(options);
   } catch (error) {
     throw new Error('Fatal error: unable to initialize Coveo Headless: ' + error);
   }
 }
 
 /**
  * 
- * @param {import("coveo").SearchEngineConfiguration} config The Headless configuration options for the specified engine ID.
+ * @param options The Headless options for the specified engine ID.
  * @param {string} engineId The id of the engine.
  * @param element The Lightning element to use to load dependencies.
  */
-function setEngineConfiguration(config, engineId, element) {
-  if (window.coveoHeadless && window.coveoHeadless[engineId] && window.coveoHeadless[engineId].config.isResolved) {
-    throw new Error(`Attempted to overwrite configuration for engine: ${engineId}`);
+function setEngineOptions(options, engineConstructor, engineId, element) {
+  if (window.coveoHeadless && window.coveoHeadless[engineId] && window.coveoHeadless[engineId].options.isResolved) {
+    console.warn(`overwriting options for engine: ${engineId}`);
```

Comment:
this logs instead of throws, fixing this: https://coveord.atlassian.net/browse/SFINT-3907